ingest/bq_ingest_arrays.sh

```diff
@@ -18,8 +18,30 @@ if [ $# -eq 5 -a "$5" = "--reprocess" ]; then
   REPROCESS=true
 fi
 
-let PARTITION_START=($DIR_ID-1)*4000+1
-let PARTITION_END=$PARTITION_START+3999
+if [ $# -eq 6 -a "$5" = "--reprocess" ]; then
+  REPROCESS=true
+fi
+
+if [ $# -eq 5 -a "$5" = "--compressed" ]; then
```

Comment:
There's got to be a better way to add another boolean argument in a bash script, right? Any thoughts?

Reply:
Nevermind, rebasing fixed it (until we want to add another flag)